src/themes.scss

```diff
@@ -0,0 +1,120 @@
+// Themes
+
+$themes: (
+  default: (
+    bg: $white-pure,
+    fg: map-get($gray, 800),
+    alt-accent-lightest: map-get($blue, lightest),
+    alt-accent-lighter: map-get($blue, lighter),
+    alt-accent-light: map-get($blue, light),
+    alt-accent: map-get($blue, themed),
+    main-accent: $teal,
+    alt-accent-baby: map-get($blue, baby),
+    red-light: map-get($red, light),
+    red-lighter: map-get($red, lighter),
+    gray-shadow: map-get($gray, shadow),
+    white-themed: $white-pure,
+    black-themed: $black-pure,
+    gray-lightest: map-get($gray, lightest),
+    gray-lighter: map-get($gray, 200),
+    gray-light: map-get($gray, 300),
+    gray-dark: map-get($gray, 800),
+    button-border-color: $black-pure,
+    form-control-border: $black-pure,
+    input-group-text-border: #ced4da,
+    select-border: map-get($gray, 300),
+    select-box-top: map-get($gray, lightest),
+    select-box-bottom: map-get($gray, 200),
+    select-box-top-bevel: $white-pure,
+    table-border: map-get($gray, 300),
+    card-bg: $white-pure,
+    card-border: #dfdfdf,
+    card-footer-bg: #f7f7f7,
+    toast-fg: #6c757d,
+    toast-bg: $white-pure,
+    toast-close: $black-pure
+  ),
+  dark: (
+    bg: map-get($gray, darkest),
+    fg: map-get($gray, 200),
+    alt-accent-lightest: #121925,
+    alt-accent-lighter: #0f1e38,
+    alt-accent-light: #1761da,
+    alt-accent: #1959c9,
```

Comment:
I wasn't sure if I should add variables / include them in the color map for colors that occur only once.